app/backend/src/couchers/db.py

```diff
@@ -235,3 +235,9 @@ def can_moderate_node(session, user_id, node_id):
     return _can_moderate_any_cluster(
         session, user_id, [cluster.id for _, _, _, cluster in get_node_parents_recursively(session, node_id)]
     )
+
+
+def timezone_at_coordinate(session, geom):
+    area = session.query(TimezoneArea.tzid).filter(func.ST_Contains(TimezoneArea.geom, geom)).one_or_none()
+    if area:
+        return area.tzid
```

Comment:
```suggestion
        return area.tzid
    return None
```
It is good to be explicit if the function is returning something.